project-base/tests/App/Acceptance/acceptance/PageObject/Front/LoginPage.php

```diff
@@ -25,9 +25,16 @@ public function login()
         $this->tester->waitForAjax();
     }
 
-    public function checkUserLogged(): void
+    /**
+     * @param string|null $fullName
+     */
+    public function checkUserLogged(?string $fullName = null): void
     {
-        $this->tester->see(static::DEFAULT_USER_NAME);
+        if ($fullName === null) {
+            $fullName = static::DEFAULT_USER_NAME;
+        }
+
+        $this->tester->see($fullName);
```

Comment:
_nitpick_ 🤓 

what about 
`$this->tester->see($fullName ?? static::DEFAULT_USER_NAME);`

Reply:
ok